.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,8 +1,8 @@
 ---
-name: 'Improvement Template '
-about:
+name: Improvement template
+about: Serves as a template for improvement issues.
 title: ''
-labels: Improvement
+labels: ''
```

Comment:
why did you remove the "improvement" label?

Reply:
sounds like i did it by mistake. i will fix it now